packages/core/src/test/amazonq/util/workspaceUtils.test.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import path from 'path'
+import { getWorkspaceForFile, getWorkspacePaths } from '../../../shared/utilities/workspaceUtils'
+import { getTestWorkspaceFolder } from '../../../testInteg/integrationTestsUtilities'
+
+describe('getWorkspace utilities', function () {
```

Comment:
was this test file intentionally placed in amazonq/util/ instead of shared/utilities/ ?